### DISCUSSION TITLE
Type issue with TS 5.0.4 and unplugin-vue-router

### DISCUSSION THREAD

**Discussion by LeaderbotX400** [posva/unplugin-vue-router#152] [Questions and Help]
![image](https://user-images.githubusercontent.com/34589843/233847143-3b7bdeaf-45d7-4fd0-b5e4-a437b02503d4.png)
![image](https://user-images.githubusercontent.com/34589843/234998546-5435569a-8e12-43ad-a14b-b2549b0be024.png)

Packages
- unplugin-vue-router: 0.5.5
- typescript: 5.0.4

**Comment by posva** [comment]
I cannot reproduce
